core/src/main/java/io/seata/core/rpc/processor/README.md

```diff
@@ -0,0 +1,80 @@
+## request
```

Comment:
Please check this @slievrly , is appropriate put the document here?

Reply:
Done.